app/models/manageiq/providers/oracle_cloud/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -2,8 +2,16 @@ module ManageIQ::Providers::OracleCloud::Inventory::Persister::Definitions::Clou
   extend ActiveSupport::Concern
 
   def initialize_cloud_inventory_collections
-    %i(vms).each do |name|
-      add_collection(cloud, name)
+    add_collection(cloud, :cloud_tenants) do |builder|
+      builder.add_default_values(:ems_id => ->(persister) { persister.manager.id} )
```

Comment:
Not sure why the core cloud_tenants collection doesn't already set the ems_id, need to look into if that was intentional or not...